tests/unit/test_interpolation.py

```diff
@@ -11,12 +11,6 @@
 from swath_projector.interpolation import (
     check_for_valid_interpolation,
     EPSILON,
-    get_bilinear_information,
-    get_bilinear_results,
-    get_ewa_information,
-    get_ewa_results,
-    get_near_information,
-    get_near_results,
```

Comment:
As with the other PR, I ran everything but `black` over `--all-files`. These are things that `ruff` picked up for us.